init.sh

```diff
@@ -1,2 +1,4 @@
 #on lxplus / machines with cvmfs
 source /cvmfs/fcc.cern.ch/sw/views/releases/0.9.1/x86_64-slc6-gcc62-opt/setup.sh
```

Comment:
Does `tricktrack` have any dependency with FCC software? Or is it just to prepare a LCG-based environment?